src/cargo/core/features.rs

```diff
@@ -712,8 +712,8 @@ pub fn nightly_features_allowed() -> bool {
     if ENABLE_NIGHTLY_FEATURES.with(|c| c.get()) {
         return true;
     }
-    match &channel()[..] {
-        "nightly" | "dev" => NIGHTLY_FEATURES_ALLOWED.with(|c| c.get()),
+    match dbg!(&channel()[..]) {
+        "nightly" | "dev" => true, //NIGHTLY_FEATURES_ALLOWED.with(|c| c.get()),
```

Comment:
@Eh2406 this is the bit that gave me trouble - it seems like `NIGHTLY_FEATURES_ALLOWED` never gets initialized in tests? How would I fix that?